### DISCUSSION TITLE
Markdown math doesn't render literal percent

### DISCUSSION THREAD

**Discussion by chadmando** [community/community#31812] [Other Feature Feedback, Questions, & Ideas] [answered]
I have tried several combinations to get either an inline math ($) or math code block to render a literal percent (\%) in an equation.

Here is the inline example:

```
$250bps * (\frac{1\%}{100bps}) = 2.5\%$
```
Renders like this in Github Markdown:

$250bps * (\frac{1\%}{100bps}) = 2.5\%$

However, the same line renders correctly in both VS Code preview and the MathJax [demo site](https://www.mathjax.org/#demo) as: 
(I am only mentioning the MathJax demo site because I understand that is the rendering engine)

<img width="446" alt="mathjax_render" src="https://user-images.githubusercontent.com/20407042/188245911-92fea8ad-417e-4732-943e-f06a2117d356.png">

**Comment by jfullstackdev** [comment, marked as answer]
$250bps * (\frac{1\\%}{100bps}) = 2.5\%$

```
$250bps * (\frac{1\\%}{100bps}) = 2.5\%$
```

you just need to put another escape character

**Reply by chadmando** [reply]
That did it! Thank you.

**Reply by dpvc** [reply]
This is due to a failure of GitHub's handling of backslashes in math mode (it doubles the ones before letters automatically, but fails to double the ones before non-letter characters).  You can certainly work around that this way for now, but if they ever fix the problem, your equations will suddenly get linebreaks at those locations, and the percent sign will cause the rest of the expression to become a comment.  Just a heads up so that when they fix the problem (I hope they do), you can go back and edit the expressions where you did that.

**Reply by Ten0** [reply]
this breaks rendering in any other viewer (e.g. VSCode's markdown preview) :(

**Reply by andrewvaughan** [reply]
@Ten0 Yep, I just noticed this too, which is unfortunate. Either the renderers have to adopt GitHub's (bug) anti-pattern or GitHub has to fix this bug in their renderer.

@dpvc already did the debugging for GitHub; just up to them to assign priority to it... it definitely seems like a 30-60 minute fix for a pretty significant rendering error... at least for those of us that do any data science on GitHub.

CC @kdaigle

**Comment by nschloe** [comment]
This is basically the same issue as https://github.com/community/community/discussions/16993. Backslashes aren't handled correctly in dollar-math.

_Edit:_ Added this to my math bug overview. https://github.com/nschloe/github-math-bugs

**Comment by nschloe** [comment]
The proper fix is to use the new backtick-delimited inline math syntax:

```markdown
$`250bps * (\frac{1\%}{100bps}) = 2.5\%`$
```

> $`250bps * (\frac{1\%}{100bps}) = 2.5\%`$

**Reply by Ten0** [reply]
Is this official anywhere else but GitHub? e.g. on VSCode's markdown preview it seems to display the backticks.

**Reply by nschloe** [reply]
That's more of a shortcoming of VSCode. GitHub and GitLab both use the backtick syntax, and imho the fact that GitHub started off with the non-backticked syntax was a huge mistake. The syntax is broken.
